operations/enterprises.md

```diff
@@ -694,6 +694,97 @@ Adds a new task to the enterprise, optionally to a specified department.
 | --- | --- | --- | --- |
 | `TaskId` | string | required | Unique identifier of added task. |
 
+## Get all tasks
+
+Returns all tasks of the enterprise, filtered by identifiers or other filters.
```

Comment:
```suggestion
Returns all bills, possibly filtered by customers, identifiers and other filters.
```

Reply:
I incorrectly copied different sentence. The important is to use `possibly`  word, to be consistent